mobile/android/base/android_strings.dtd

```diff
@@ -111,15 +111,19 @@
 <!ENTITY pref_category_vendor2 "&vendorShortName; &brandShortName;">
 <!ENTITY pref_category_vendor_summary2 "O programie &brandShortName;, najczęściej zadawane pytania, opinie">
 <!ENTITY pref_category_datareporting "Udostępnianie danych">
+<!ENTITY pref_category_logins "Dane logowania">
 <!ENTITY pref_learn_more "Więcej informacji">
 <!ENTITY pref_category_installed_search_engines "Zainstalowane wyszukiwarki">
 <!ENTITY pref_category_add_search_providers "Dodawanie wyszukiwarek">
 <!ENTITY pref_category_search_restore_defaults "Przywróć wyszukiwarki">
 <!ENTITY pref_search_restore_defaults "Przywróć domyślne">
 <!ENTITY pref_search_restore_defaults_summary "Przywróć domyślne">
-<!ENTITY pref_search_hint "Rada: Możesz dodać każdą witrynę do swojej listy wyszukiwarek, przytrzymując na niej pole wyszukiwania, a następnie stukając ikonę &formatI;.">
+<!ENTITY pref_search_hint2 "Rada: Możesz dodać każdą witrynę do swojej listy wyszukiwarek, przytrzymując na niej pole wyszukiwania, a następnie dotykając ikony &formatI;.">
 <!ENTITY pref_category_advanced "Zaawansowane">
 <!ENTITY pref_category_advanced_summary2 "Przywracanie kart, wtyczki, narzędzia dla programistów">
+<!ENTITY pref_category_notifications "Powiadomienia">
+<!ENTITY pref_content_notifications "Aktualizacje stron">
+<!ENTITY pref_content_notifications_summary2 "Nowości z obsługiwanych stron">
```

Comment:
"Aktualizacje stron" (`pref_content_notifications`) and "Nowości z obsługiwanych stron" (`pref_content_notifications_summary2`) - this is misleading. Maybe "Powiadomienia ze stron" and "Umożliwia obsługiwanym stronom wysyłanie powiadomień systemowych" or something. It would be best to reword it also in `en-US` but it needs update in `pl` in the first place.

"Firefox Mobile: co nowego" looks bad as an option label, maybe "Nowości w &brandShortName;"?

And don't forget to update "Nowe funkcje, aktualizacje stron" (`pref_category_notifications_summary`) accordingly.